.github/scripts/apply_review.py

```
@@ -276,6 +296,76 @@ def run(self) -> bool:
+
+        for turn in range(MAX_TURNS):
+            log.info(f"Turn {turn + 1}/{MAX_TURNS} model={OPUS_MODEL} (Anthropic)")
+            try:
+                resp = client.messages.create(
+                    model=OPUS_MODEL,
+                    max_tokens=4096,
+                    system=system_content,
+                    tools=anthropic_tools,  # type: ignore[arg-type]
+                    messages=messages,      # type: ignore[arg-type]
+                )
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)